src/lightning_app/core/queues.py

```diff
@@ -96,6 +97,10 @@ def get_api_delta_queue(self, queue_id: Optional[str] = None) -> "BaseQueue":
         queue_name = f"{queue_id}_{DELTA_QUEUE_CONSTANT}" if queue_id else DELTA_QUEUE_CONSTANT
         return self.get_queue(queue_name)
 
+    def get_app_status_queue(self, queue_id: Optional[str] = None) -> "BaseQueue":
```

Comment:
Let's not add another queue, this is hard to handle properly in the cloud with the launcher.